src/optimistic-rollups-dodao-academy.yaml

```diff
@@ -303,20 +306,20 @@ steps:
     name: Evaluation
     stepItems:
       - answerKeys:
-          - c038a5a7
+          - 9b0493fa
         choices:
-          - content: A
+          - content: State transition's pre-state
             key: 1a95824f
-          - content: B
+          - content: The state transition itself
             key: c038a5a7
-          - content: C
+          - content: Executor's asserted post-state
             key: f00d64fb
-          - content: D
+          - content: All of the above
             key: 9b0493fa
-        content: Q3
+        content:  The list of inclusion proofs in One-round fraud proof for quick and easy fraud detection are?
```

Comment:
The question and option as not easy for anyone new to answer. HAs very specific language